### PR TITLE
[ON HOLD] Article built-in modules

### PR DESCRIPTION
**Pull request by Julievrz**
This article aims at highlighting all the work that has been done around built-in modules:

- Design of new logos
- Update of the built-in module pages on Addons
- Creation of a brochure in 8 languages

### CODE REVIEW

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+subtitle: "This article will answer all your questions about built-in modules"
+date: 2020-01-18 
+authors: [JulieVarisellaz]
+icon: icon-world
+
+---
+
+Do you know that PrestaShop 1.7.7 comes with 88 built-in modules? That is 88 ways for merchants to start their online store and empower their business, directly from their back office! :fire:
+
+In this article, we will try to answer all your questions about this subject. So let’s get started!
```
Comment (ttoine) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]:
```suggestion
In this article, we will try to answer all your questions about this topic. So let’s get started!
```

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "All you need to know about built-in modules"
```
Comment (ttoine) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md:3]: What about a title telling it's an **update** about what has been done recently?

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+We are not talking about "native modules" here, as the definition of a native module is a bit more complicated. Indeed, some native modules are present in the installation zip, some are pre-installed and some others are not. So, to make it easier, we have chosen to talk about built-in modules in this article.
+
+## Where can I find the list of all built-in modules?
+
+There was a time where it was difficult to find the list of all built-in modules. You could have a part of it directly in the back office but half of the information was missing and most of it has not been updated (or worse, inherited from the 1.6 version). In the end, it was not helpful at all.
+
+Dear PrestaShop users, those days are over! 
+
+Our core product team took some time to test all modules, screenshot them, accurately categorize them, write explicit and complete product information, translate it into eight languages, create special Addons pages for them and gather it all in a dedicated brochure available  [here](https://www.prestashop.com/fr/ressources).
```
Comment (ttoine) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]:
```suggestion
The PrestaShop company's core product team took some time to test all modules, screenshot them, accurately categorize them, write explicit and complete product information, translate it into eight languages, create special Addons pages for them and gather it all in a dedicated brochure available  [here](https://www.prestashop.com/fr/ressources).
```
let's be clear who did this to the open source project.

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "All you need to know about built-in modules"
+subtitle: "This article will answer all your questions about built-in modules"
+date: 2020-01-18
```
Comment (Julievrz) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]: I tried to add a time, but I don't know why it did not work.

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+![New logos](/assets/images/2021/01/build-article-modules-logos-january21.png)
+
+You can also have a look at the [GitHub issue](https://github.com/PrestaShop/PrestaShop/issues/15858).
+
+## What if I want to know more about a module in particular?
+
+To know more about a specific module (whether it is a built-in module or not) you should visit the [PrestaShop Official Addons Marketplace](https://addons.prestashop.com/en/).
+
+Our product managers have redesigned the built-in module pages on the marketplace to make them clearer and more understandable.
```
Comment (ttoine) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md:97]: could you please add a link to this page?

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+You can also have a look at the [GitHub issue](https://github.com/PrestaShop/PrestaShop/issues/15858).
+
+## What if I want to know more about a module in particular?
+
+To know more about a specific module (whether it is a built-in module or not) you should visit the [PrestaShop Official Addons Marketplace](https://addons.prestashop.com/en/).
+
+Our product managers have redesigned the built-in module pages on the marketplace to make them clearer and more understandable. 
+
+Please note that you cannot download built-in modules on the Addons marketplace, as they are already available in both Modules sections of your back office.
```
Comment (ttoine) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]:
```suggestion
Please note that you cannot download built-in modules on the Addons marketplace, as they are already available in the back office of your shop. 
```

**Review by ttoine — CHANGES_REQUESTED** [review]

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md (hunk 5 above)
Comment (Julievrz) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md:97]: Well, this is not just one page. There are as many pages as there are built-in modules. All these pages are available on the Addons Marketplace.
I plan to add the links to each page on the brochures, so access to those pages become easier for the merchants. 😊

Comment (ttoine) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md:97]: ok, then, maybe you can tell something about that in the article.
thanks ;-)

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md (hunk 6 above)
Comment (Julievrz) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]: Isn't it better to let them know exactly where to find them in their back office?

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+Best customers:
+Enrich your stats, add a list of the best customers to the dashboard
+
+![New logos](/assets/images/2021/01/build-article-modules-logos-customer-follow-up-january21.png)
+
+Customer follow-up: 
+Offer your customers a quick follow-up with daily customized e-mails to send
+
+![New logos](/assets/images/2021/01/build-article-modules-logos-customer-official-gdpr-january21.png)
```
Comment (Julievrz) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]:
```suggestion
![New logos](/assets/images/2021/01/build-article-modules-logos-official-gdpr-january21.png)
```

**Comment by Julievrz** [comment]
If someone could tell me how to reduce Tristan and Louise's pictures, I would be very grateful. :)

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md (hunk 2 above)
Comment (Julievrz) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md:3]: Let me know what you think of this new subtitle :)

**Comment by Julievrz** [comment]
If someone could explain to me how to align the images with the text, I would be very grateful. I would like it to look like this:

image

I already tried to reduce the size following an HTML tutorial, but I'm not sure that the result is great 😅

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md (hunk 6 above)
Comment (LouiseBonnard) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]: Hi @Julievrz, what do you think about telling users the modules cannot be downloaded on Addons but, if need be, via the dedicated GitHub repository?

Comment (Julievrz) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]: Good idea. But if understand well what you mean, there are as many repositories as they are built-in modules, right? So they would have to find them themselves?

Comment (LouiseBonnard) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]: Yep, just like you did when you went through the update of all the logo and description on GitHub. By the way, you should mention that in this post as well, you did a great job! Build readers are supposed to know about GitHub repositories but you can still add a link. :-)

**Comment by Julievrz** [comment]
> If someone could explain to me how to align the images with the text, I would be very grateful. I would like it to look like this:
> 
> image
> 
> I already tried to reduce the size following an HTML tutorial, but I'm not sure that the result is great 😅

@ttoine @matks Any idea? ☺️

**Comment by matks** [comment]
> > If someone could explain to me how to align the images with the text, I would be very grateful. I would like it to look like this:
> > image
> > I already tried to reduce the size following an HTML tutorial, but I'm not sure that the result is great 😅
> 
> @ttoine @matks Any idea? ☺️

Markdown does not provide out-of-the box images alignment.

For such features you'll need to rely either on custom CSS, custom HTML or on Jekyll (which powers this website, [example](https://mmistakes.github.io/jekyll-theme-basically-basic/markup/markup-image-alignment/)) features but you need explore :/

**Comment by Julievrz** [comment]
> > > If someone could explain to me how to align the images with the text, I would be very grateful. I would like it to look like this:
> > > image
> > > I already tried to reduce the size following an HTML tutorial, but I'm not sure that the result is great 😅
> > 
> > 
> > @ttoine @matks Any idea? ☺️
> 
> Markdown does not provide out-of-the box images alignment.
> 
> For such features you'll need to rely either on custom CSS, custom HTML or on Jekyll (which powers this website, [example](https://mmistakes.github.io/jekyll-theme-basically-basic/markup/markup-image-alignment/)) features but you need explore :/

Hi @matks,
Thank you but unfortunately, I was unable to do it (I tried many times 😭). So, I replaced it with a picture. I know it is not very good for SEO, but I didn't want this to delay the publication. The article is ready for review now, let me know when I can publish it. ☺️ Thank you!

**Comment by ttoine** [comment]
Julie, you should not really do that, it will not be easily responsive. It might be nice looking on a pc with a large screen, but not a on mobile device with a vertical screen or a small screen.

On blog articles, in general, the best practice is usually to put an image centered between 2 paragraphs. Because you don't know what device will be used to read the content.

**Comment by Julievrz** [comment]
> Julie, you should not really do that, it will not be easily responsive. It might be nice looking on a pc with a large screen, but not a on mobile device with a vertical screen or a small screen.
> 
> On blog articles, in general, the best practice is usually to put an image centered between 2 paragraphs. Because you don't know what device will be used to read the content.

Hi @ttoine, in that case, I suggest getting rid of the images and write the quote and the name after it. :) Because the image alone with the text under it is not very aesthetic.

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+In this article, we will try to answer all your questions about this topic. So let’s get started!
+
+## What is a built-in module exactly?
+
+A built-in module is a module that was developed by PrestaShop and that is available in your back office either in the Module Manager or in the Module Catalog. 
+
+We are not talking about "native modules" here, as the definition of a native module is a bit more complicated. Indeed, some native modules are present in the installation zip, some are pre-installed and some others are not. So, to make it easier, we have chosen to talk about built-in modules in this article.
+
+To know more about native modules, you can check the [documentation](https://devdocs.prestashop.com/1.7/development/native-modules/).
```
Comment (LouiseBonnard) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]:
```suggestion
To know more about native modules, you can check the [documentation](https://devdocs.prestashop.com/1.7/development/native-modules/).

### New native modules distribution

Following an opinion poll carried out through the [User Club](https://www.prestashop.com/en/club) and the usage studied along with the project's maintainers, we made the decision to redefine the current native modules distribution. Some have not been updated for years, some did not work with PrestaShop 1.7, and others could be merged into a single module, this is why a small number of modules have just been archived. It means that they are still available on GitHub (although they have been removed from the organization) but the Core team no longer maintains them. You still can fork it if you need it for your shop.
```

Comment (LouiseBonnard) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]: @marionf, @eternoendless, feel free to modify this part if need be, thanks!

Comment (marionf) [reply news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]: I just added that they can be forked :)

File: news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+### New native modules distribution
+
+Following an opinion poll carried out through the [User Club](https://www.prestashop.com/en/club) and the usage studied along with the project's maintainers, we made the decision to redefine the current native modules distribution. Some have not been updated for years, some did not work with PrestaShop 1.7, and others could be merged into a single module, this is why a small number of modules have just been archived. It means that they are still available on GitHub (although they have been removed from the organization) but the Core team no longer maintains them. You still can fork it if you need it for your shop.
```
Comment (LouiseBonnard) [inline news/_posts/2021-01-18-all-you-need-to-know-about-built-in-modules.md]:
```suggestion
Following an opinion poll carried out through the [User Club](https://www.prestashop.com/en/club) and the usage studied along with the project's maintainers, we made the decision to redefine the current native modules distribution. Some have not been updated for years, some did not work with PrestaShop 1.7, and others could be merged into a single module, this is why a small number of modules have just been archived. It means that they are still available on GitHub (although they have been removed from the organization) but the Core team no longer maintains them. You still can fork them if you need them for your shop.
```

**Comment by matks** [comment]
Hello I close this PR as there was no activity for almost a year.

It does not mean we will not merge it: when it's possible to work again on it please reopen. But I close it so it does not appear in the list of items "to be reviewed".

**Pull request closed without merging**